behat.yml.dist

```diff
@@ -29,14 +31,15 @@ default:
               Project creation: 'node/add/oe_project'
               Publication creation: 'node/add/oe_publication'
               Call for tenders creation: 'node/add/oe_call_tenders'
+              Call for proposals creation: 'node/add/oe_call_proposals'
               Organisation creation: 'node/add/oe_organisation'
               AV Portal photo creation: 'media/add/av_portal_photo'
               document creation: 'media/add/document'
               interface translation: 'admin/config/regional/translate'
   extensions:
     Drupal\MinkExtension:
       goutte: ~
-      ajax_timeout: 10
+      ajax_timeout: 30
```

Comment:
There were a discussion about ajax_timeout. If Drone works well with `ajax_timeout: 10` then it should be left as is. You can change this value locally in `behat.yml` if this timeout is not enough. If Drone fails with `ajax_timeout: 10` then it's better to identify where performance issue is.

Reply:
I don't need to change the `ajax_timeout` setting in my local environment. I increased the value in order the test to pass while the project is built by drone.

Reply:
lets investigate further